cypress/integration/collections_spec.js

```diff
@@ -117,5 +117,24 @@ describe('Dashboard Collections Page', () => {
       cy.contains('.heading--xlarge', 'Collections');
       cy.contains('table tbody tr a', name).should('not.exist');
     });
+
+    it('should fail deleting a collection with an associated rule', () => {
+      const name = 'MOD09GQ';
+      const version = '006';
+
+      cy.visit(`/#/collections/collection/${name}/${version}`);
+
+      // delete collection
+      cy.contains('button', 'Delete').click();
+      cy.contains('button', 'Confirm').click();
+
+      // error should be displayed indicating that deletion failed
+      cy.get('.error__report');
```

Comment:
add assertion to check the error message

Reply:
I actually intentionally did not do that because it seemed kinda brittle. As long as these tests run against the fake API, it should be safe, though in the future they might run against our actual API?